test/local/routes/signin-utils.js

```diff
@@ -0,0 +1,128 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Can you make the path of this file match the path it's testing? So `test/local/routes/utils/signin.js`?